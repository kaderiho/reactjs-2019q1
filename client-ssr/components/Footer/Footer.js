"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;

var _react = _interopRequireDefault(require("react"));

var Footer = function Footer() {
  return _react.default.createElement("footer", null, _react.default.createElement("a", {
    href: "/",
    className: "logoLink"
  }, "netflixroulette"));
};

var _default = Footer;
exports.default = _default;