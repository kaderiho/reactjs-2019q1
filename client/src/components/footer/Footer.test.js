import React from 'react';
import { shallow , mount } from 'enzyme';
import Footer from './Footer';

describe('Footer component', () => {
    it('Should render correctly', () => {
        const wrapper = shallow(<Footer />);
        expect(wrapper).toMatchSnapshot();
    });

    it('Should contain link element', () => {
        const wrapper = mount(<Footer />);
        expect(wrapper.exists('.logoLink')).toEqual(true);
    });

    it('Link should has correct href', () => {
        const wrapper = mount(<Footer />);

        expect(wrapper.exists({ href: '/'})).toEqual(true);
    });
});