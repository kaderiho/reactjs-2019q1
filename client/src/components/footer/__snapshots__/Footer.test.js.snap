// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Should render correctly 1`] = `
<footer>
  <a
    className="logoLink"
    href="/"
  >
    netflixroulette
  </a>
</footer>
`;
